cartographer_ros/CMakeLists.txt

```diff
@@ -57,43 +52,39 @@ if(DEFINED urdfdom_headers_VERSION)
 endif()
 
 include_directories(
-  ${urdfdom_headers_INCLUDE_DIRS}
+  include
+  "."
```

Comment:
Yeah, we need because a lot of the files do things like:

#include "cartographer_ros/msg_conversion.h"